oio/xcute/common/worker.py

```diff
@@ -57,8 +58,10 @@ def process_beanstalkd_job(self, beanstalkd_job):
             tasks_run_time = ratelimit(
                     tasks_run_time, tasks_per_second)
 
+            reqid = job_id + request_id('-')
```

Comment:
Shouldn't we simply use `request_id(job_id + '-')` instead?

Reply:
The function returns 32 bytes and `job_id` already measures 32 bytes.